content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/amp.html

```diff
@@ -0,0 +1,37 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<amp-accordion data-sly-use.accordion="com.adobe.cq.wcm.core.components.models.Accordion"
+     class="cmp-accordion"
+     data-cmp-is="accordion"
+     data-sly-attribute.expand-single-section="${accordion.singleExpansion}">
+      <section data-sly-test="${accordion.items.size > 0}"
+        data-sly-repeat.item="${accordion.items}"
+        class="cmp-accordion__item"
+        data-cmp-hook-accordion="item"
```

Comment:
all `data-cmp-hook-<component>` attributes can be removed in the amp renderings, they identify sub components for widgets in the standard rendering. Same comment for the Form Text, Search, Tabs, Carousel and Image components.